src/openassetio-core/include/openassetio/managerApi/ManagerInterface.hpp

```diff
@@ -534,6 +534,27 @@ class OPENASSETIO_CORE_EXPORT ManagerInterface {
    * @}
    */
 
+  /**
+   * @name Batch element error handling
+   *
+   * @{
+   */
+  /**
+   * Callback signature used for an unsuccessful operation on an
+   * element in a batch.
+   *
+   * This should be called for errors that are specific to a particular
+   * reference in a batch. Exceptions can be thrown to indicate a
+   * whole-batch error.
+   *
+   * The appropriate error code should be used for these errors. See
+   * @fqref{BatchElementError.ErrorCode} "ErrorCode".
```

Comment:
[consider][minor] "The appropriate error code should be used for these errors." is a bit of an empty statement.

```suggestion
   * See @fqref{BatchElementError.ErrorCode} "ErrorCode". for
   * appropriate error codes.
```

Reply:
Not technically part of this PR (just moved that code up), but good suggestion, updated in 725a6bbb

Reply:
Ah, sorry, I know it's annoying to get comments on not your stuff, didn't realise, thanks for changing.